packages/utils/src/lib/reports/__snapshots__/generate-md-report.integration.test.ts.snap

```diff
@@ -15,14 +15,18 @@ exports[`generateMdReport > should contain all sections when using the fixture r
 
 Performance metrics [📖 Docs](https://developers.google.com/web/fundamentals/performance)
 
-🟢 Score:  **92**
+🟡 Score:  **74**
 - 🟢 Performance (_Lighthouse_)
   - 🟩 [Total Blocking Time](#total-blocking-time-lighthouse) - **0 ms**
   - 🟨 [Largest Contentful Paint](#largest-contentful-paint-lighthouse) - **1.5 s**
   - 🟩 [Cumulative Layout Shift](#cumulative-layout-shift-lighthouse) - **0**
   - 🟨 [First Contentful Paint](#first-contentful-paint-lighthouse) - **1.2 s**
   - 🟩 [Speed Index](#speed-index-lighthouse) - **1.2 s**
 
+- 🔴 ESLint performance rules (_ESLint_)
+  - 🟥 [Disallow unused variables](#disallow-unused-variables-eslint) - **1 warning**
+  - 🟥 [Enforce a maximum number of lines of code in a function](#enforce-a-maximum-number-of-lines-of-code-in-a-function-eslint) - **1 warning**
```

Comment:
I would prefer to keep the mock report as is, this configuration is less meaningful (e.g. why should unused variables impact performance score?). I realize the mock report doesn't cover your use case of sorting multiple groups, but I think it would be better to tests these different cases at source (i.e. in `sorting.unit.test.ts`).

Reply:
There is no other test that currently tests the full output of `generateMdReport` (the E2E test is very limited). So in order to check that multiple groups are correctly displayed there, at least two of them have to exist. At the moment, there is at most one which is not sufficient. I will be happy to hear better suggestions in regards to the test data.

Group sorting that involves its audits being sorted has test cases added in `utils.unit.test.ts`, namely for `getSortableGroupByRef` and `compareCategoryAuditsAndGroups`.

Reply:
If we were integrate this into the realistic test data, then I'd suggest the following changes:
- replace [no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars) (unrelated to performance) with [react/jsx-key](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md) (101 of React performance)
- give the ESLint group weight 0, so it doesn't impact the category score